addons/knowledge/models/knowledge_article.py

```diff
@@ -748,29 +758,47 @@ def get_field_info(article, field_name):
     # Permission and members handling methods
     # ---------------------------------------
 
-    def set_article_permission(self, permission):
+    def set_internal_permission(self, permission):
```

Comment:
as those methods are not called directly by rpc, but by controller instead, make them private